README.md

```diff
@@ -15,8 +15,21 @@ This README is, itself, incomplete! I'd love some help adding resources for
 those folks who are new to Git/GitHub. Please feel free to contribute to this
 portion of the project as well!
 
+## How to Submit a Pull Request
+1. Navigate to the project's main page (i.e. https://github.com/csteinert/friends-of-gdic).
```

Comment:
While I appreciate the effort to come up with this list, it's a bit too specific to the manual editing process of editing files on GitHub. I worry that it'll confuse people, since it isn't necessary to edit via GitHub in order to submit a PR. The two links on lines 27 and 28 are what I would expect for this section. Can you pare your changes down to those two lines? Thanks!